pkg/minikube/machine/start.go

```diff
@@ -233,17 +234,36 @@ func postStartValidations(h *host.Host, drvName string) {
 		return
 	}
 
+	if viper.GetBool("force") {
+		return
+	}
+
 	// make sure /var isn't full,  as pod deployments will fail if it is
```

Comment:
Trim extra space.

Reply:
which extra space are you referring to?
the empty line on 240?
I think it makes sense to add that empty line because of the comment that is about to follow, imo easier to catch up on it that way